Makefile

```diff
@@ -415,14 +415,14 @@ check-diff:
 ################################################################################
 .PHONY: check-proto-version
 check-proto-version: ## Checking the version of proto related tools
-	@test "$(shell protoc --version)" = "libprotoc 3.21.1" \
-	|| { echo "please use protoc 3.21.1 (protobuf 21.1) to generate proto, see https://github.com/dapr/dapr/blob/master/dapr/README.md#proto-client-generation"; exit 1; }
+	@test "$(shell protoc --version)" = "libprotoc 3.21.12" \
+	|| { echo "please use protoc 3.21.12 (protobuf 21.12) to generate proto, see https://github.com/dapr/dapr/blob/master/dapr/README.md#proto-client-generation"; exit 1; }
```

Comment:
should we have a variable to hold the proto version to avoid repetition?

Reply:
I am moving this change to a separate PR and will do that change there